src/config.rs

```diff
@@ -39,3 +39,10 @@ pub fn zo_maxage() -> Result<Rank> {
         None => Ok(1000.0),
     }
 }
+
+pub fn zo_exclude_dirs() -> Option<Vec<PathBuf>> {
```

Comment:
Rather than `Option<Vec<PathBuf>>`, why not return an empty `Vec` if the environment variable is not set? It simplifies the code, and shouldn't have much of an impact on performance.

Also, a minor nitpick: since there isn't a logical way to order the functions in this module, could you order them alphabetically instead? I think it would make things easier once the number of environment variables increases.

Reply:
Roger. I'll do the same for the README, if you don't mind.